### DISCUSSION TITLE
Integrate solid-app-router with store?

### DISCUSSION THREAD

**Discussion by tslocke** [solidjs/solid#976] [Q&A] [answered]
Is there any library or recommended approach to integrate the router with the store, so e.g. I can get current params as store properties?

I used to use something like this in Vue-land and it was pretty convenient.

Thanks

**Comment by Exelord** [comment, marked as answer]
Hi Tom :) Im not in vue-land, but could try to help. You can access router params through `useParams()`  which is returning immutable store. If that does not help, please let me understand your issue better and share with me more details :)

**Reply by tslocke** [reply]
Thanks - I made a doc PR to have this clarified in the readme.
